packages/types/src/element/element.ts

```diff
@@ -65,4 +75,9 @@ export interface Element extends ElementBox {
   groupId?: string;
   border?: Border;
   borderRadius?: BorderRadius;
+  pageAttachment?: PageAttachment;
```

Comment:
Elements can't have page attachments, only pages can.

Reply:
Fixed in https://github.com/GoogleForCreators/web-stories-wp/pull/12382/commits/a1cea5e2504a0731e448b3e5895b18eb6562016d